tests/test_io.py

```diff
@@ -1,15 +1,9 @@
 from __future__ import annotations
 
 import os
-import unittest
+from pathlib import Path
 
 import pytest
-
-try:
-    from pathlib import Path
-except ImportError:
-    Path = None  # type: ignore
```

Comment:
I think it should be safe to remove this guard against `pathlib` now that we're requiring Python > 3.9. Let me know if there's other reasons that I'm not aware of.
https://github.com/materialsvirtuallab/monty/blob/feb6e8323eb0e074ee5f85838fe67e24682376eb/pyproject.toml#L12